sagemaker-python-sdk/pytorch_batch_inference/sagemaker_batch_inference_torchserve.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
@nskool  we need to remove cell 11, where it send requests one by one

Reply:
Done